src/hotspot/share/cds/filemap.cpp

```diff
@@ -286,8 +275,6 @@ void FileMapHeader::print(outputStream* st) {
   st->print_cr("- compressed_class_ptrs:          %d", _compressed_class_ptrs);
   st->print_cr("- cloned_vtables_offset:          " SIZE_FORMAT_X, _cloned_vtables_offset);
   st->print_cr("- serialized_data_offset:         " SIZE_FORMAT_X, _serialized_data_offset);
-  st->print_cr("- heap_begin:                     " INTPTR_FORMAT, p2i(_heap_begin));
-  st->print_cr("- heap_end:                       " INTPTR_FORMAT, p2i(_heap_end));
```

Comment:
Maybe print the _heap_roots_offset?

Reply:
Hi Calvin, thanks for the review. I added the `_heap_roots_offset` into the debug output.